files/incremental_upload.py

```
@@ -500,6 +501,42 @@ def check_identical_samplesheets(samplesheet_1, samplesheet_2) -> bool:
+            for row in reader:
+                if not row:
+                    continue
+                key = row[0].strip().strip('"').casefold()
+                if key in ("experiment", "experiment name", "experiment name:"):
+                    # Find first non-empty value in the row (skip index 0 which is the key)
+                    for value in row[1:]:
+                        value = value.strip().strip('"')
+                        if value:  # First non-empty value
+                            return value
+                    return None
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)